### DISCUSSION TITLE
What breed of dog is Cujo?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#242] [Galaxy Trivia] [answered]
![cujo](https://user-images.githubusercontent.com/5120290/224054512-ea07b9fe-8268-48f7-a226-1c47bdd29414.png)

**Comment by thuongtruong1009** [comment, marked as answer]
Cujo is a fictional character in a horror novel and subsequent film adaptation by Stephen King. He is portrayed as a St. Bernard dog who becomes infected with rabies and goes on a violent rampage. However, it's important to note that in reality, the breed of dog (St. Bernard) does not determine aggressiveness or violent behavior, and rabies can affect any mammal, including humans.
